graphql/README.md

```diff
@@ -0,0 +1,22 @@
+# GraphQL Data Connector
+
+Follow these steps to get started with GraphQL as a Data Connector.
+With quickstart uses Github's GraphQL API as an example.
```

Comment:
Quickstarts are separate from samples.  The scenario we are unlocking with quickstarts is: If you already have a graphql API you'd like to use with spice, you can use this to quickly get started.

Reply:
So we can make this example a sample.  And I like the idea.  So lets move this to samples and make a proper quickstart.